.gitignore

```diff
@@ -46,6 +46,12 @@ coverage.xml
 *.cover
 .hypothesis/
 .pytest_cache/
+cov.xml
+root/
+edge-agent-ca/
```

Comment:
For this kind of test output, we should delete them so that we have a clean state every time we start a new test.

Reply:
You can use the tests/output folder.

Reply:
cov.xml should be ignore because this will be reviewed after tested. Other files will be removed after tested.